tools/rebasehelpers/util/git.go

```diff
@@ -98,12 +98,15 @@ func (c Commit) GodepsReposChanged() ([]string, error) {
 
 type File string
 
-func (f File) HasGodepsChanges() bool {
+func (f File) HasGodepsChanges(includingGodeps bool) bool {
```

Comment:
I went with `HasVendoredCodeChanges` and `HasGodepsChanges` since we're already using godeps elsewhere for Godeps.json so I kept that in sync. I guess that's reasonable approach.